src/lib/__tests__/InstantSearch-test.js

```diff
@@ -392,12 +361,11 @@ describe('start', () => {
       indexName: 'indexName',
       searchClient,
       searchFunction(helper) {
-        helper.addDisjunctiveFacetRefinement('brand', 'Apple');
-        helper.search();
-      },
-      searchParameters: {
-        disjunctiveFacetsRefinements: { brand: ['Apple'] },
-        disjunctiveFacets: ['brand'],
+        const nextState = helper.state
+          .addDisjunctiveFacet('brand')
+          .addDisjunctiveFacetRefinement('brand', 'Apple');
+
+        helper.setState(nextState).search();
```

Comment:
can be done without touching the state methods, since both exist on helper "root"

Reply:
You can't call `addDisjunctiveFacet` on the root, it's only with a configuration object.